configs/prettier-config-devtools/bin/monorepo-prettier

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# run prettier inside this repo so we don't have to specify prettier as a dep
+# everywhere we specify @mongodb-js/prettier-config/devtools as a dep
+
+# Why npx? The current directory is probably in a different package, probably a
+# different monorepo rather than this one. So that's where we're executing
+# prettier from. And we can't mess with the current directory, because the
+# command-line args assume we're there. But we don't want to have to install
+# prettier everywhere so the best we can do is probably to specify prettier once
+# per monorepo at the top.
+
+npx prettier "$@"
```

Comment:
I think this is already pretty good, but if you want to avoid having prettier even at the monorepo root and make sure that you use the one that will be installed with this package, you can do something similar to [what we do in webpack-config-compass](https://github.com/mongodb-js/compass/blob/3813e8433f30460bc7d36ad98a6596be4d825197/configs/webpack-config-compass/bin/webpack.js#L36)